packages/flame_tiled/lib/src/renderable_layers/image_layer.dart

```diff
@@ -0,0 +1,62 @@
+part of '../renderable_tile_map.dart';
+
+class _ImageLayer extends _RenderableLayer<ImageLayer> {
+  final Image _image;
+  late final ImageRepeat _repeat;
+  Rect _paintArea = Rect.zero;
+
+  _ImageLayer(super.layer, super.parent, this._image) {
+    _initImageRepeat();
+  }
+
+  @override
+  void handleResize(Vector2 canvasSize) {
+    _paintArea = Rect.fromLTWH(0, 0, canvasSize.x, canvasSize.y);
```

Comment:
Not related to this PR, but here we can do this instead:
```suggestion
    _paintArea = canvasSize.toRect();
```

Reply:
Done 👍🏻